_automating-workflows/workflow-tutorial.md

```diff
@@ -122,19 +122,18 @@ If you define `previous_node_inputs`, then defining edges is optional.
 A CoT agent can use the deployed model in a tool. This step doesn’t strictly correspond to an API but represents a component of the body required by the [Register Agent API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/). This simplifies the register request and allows reuse of the same tool in multiple agents. For more information about agents and tools, see [Agents and tools]({{site.url}}{{site.baseurl}}/ml-commons-plugin/).
 
 <!-- vale off -->
-### math_tool
+### cat_index_tool
 <!-- vale on -->
 
-You can configure other tools to be used by the CoT agent. For example, you can configure a math tool as follows. This tool does not depend on any previous steps:
+You can configure other tools to be used by the CoT agent. For example, you can configure a `cat_index_tool` as follows. This tool does not depend on any previous steps:
 
 ```yaml
-- id: math_tool
+- id: cat_index_tool
   type: create_tool
   user_inputs:
-    name: MathTool
-    type: MathTool
-    description: A general tool to calculate any math problem. The action input
-      must be a valid math expression, like 2+3
+    name: CatIndexTool
+    type: CatIndexTool
+    description: A tool to retrieve index information
```

Comment:
I learned later that these descriptions are important to the quality of the tool's output and it's better to allow the default description.  So let's delete this line.  (Looks like you got all the others, just missed this one.)
```suggestion
```